pkg/agent/flowexporter/connections/conntrack_ovs.go

```diff
@@ -49,11 +50,25 @@ func NewConnTrackOvsAppCtl(nodeConfig *config.NodeConfig, serviceCIDR *net.IPNet
 	if ovsctlClient == nil {
 		return nil
 	}
-	return &connTrackOvsCtl{
+	ct := &connTrackOvsCtl{
```

Comment:
same here

Reply:
Variable ct needed here to do RunAppctlCmd call for getting maxconns.

Reply:
can't we use ovsctlClient directly here? Moving the initialization of structure to the end makes the code easy to read I think.

Reply:
Make senses. Fixed it, thanks!